keyboards/handwired/hillside/keymaps/default/README.md

```diff
@@ -0,0 +1,179 @@
+# Default Keymap
```

Comment:
please rename this file to be lower case.

Reply:
Done. Sorry I missed those.